### DISCUSSION TITLE
모든 곳에 떨어지고 딜리버리 히어로에 가게 된 이야기

### DISCUSSION THREAD

**Discussion by giscus** [Cyma-s/Cyma-s.github.io#101] [General]
# 모든 곳에 떨어지고 딜리버리 히어로에 가게 된 이야기

우아한테크코스를 수료하고 나서 많은 곳에 인턴 서류를 넣었다. 그 중에는 대학교 연합 동아리 서류도 있었는데, 결과는 다음과 같다. 당근마켓 인턴 두 곳 서류 탈락 카카오 인턴 코딩 테스트 탈락 에이블리 인턴 서류 탈락 학교에서 중개해주는 placement 인턴 서류 탈락 Nexters 서류 탈락 연이은 서류 탈락들에 어지러웠다. 나는 정말 잘한 거였을까…

https://vero.wiki/get-a-job-with-delivery-hero/

**Comment by greeng00se** [comment]
어려움 속에서도 도망가지 않고 앞으로 나아간 부분, 부족함을 인지하고 개선하려는 부분이 정말 대단하네요!
고생많았고, 축하하고, 앞으로도 응원하겠습니다~

**Reply by Cyma-s** [reply]
허브도 축하합니다 🚀 앞으로도 파이팅이에요~
많관 감사합니다 🙇‍♀️

**Comment by ezzanzzan** [comment]
오랜만에 들어왔는데 글 읽으면서 뭔가 울컥 왈칵 .. 그동안 고생 많았어 ! 앞으로 행복한 일만 가득하길 바라 ❤︎

**Reply by Cyma-s** [reply]
토리 🥹🥹🥹🥹 토리도 고생 많아써ㅠㅠㅠ 앞으로도 행복하자 !! 🥰
